cscs-checks/tools/io/cdo.py

```diff
@@ -170,11 +168,13 @@ def __init__(self, **kwargs):
         self.descr = ('verifies merging and compressing of 3 compressed '
                       'netCDF-4 files')
         self.executable = 'cdo'
-        self.executable_opts = ['-O', '-z', 'zip', 'merge',
-                                'test_echam_spectral-deflated_wind10.nc4c',
-                                'test_echam_spectral-deflated_wl.nc4c',
-                                'test_echam_spectral-deflated_ws.nc4c',
-                                'test_echam_spectral-deflated_wind10_wl_ws.nc4c']
+        self.executable_opts = [
+            '-O', '-z', 'zip', 'merge',
+            'test_echam_spectral-deflated_wind10.nc4c',
+            'test_echam_spectral-deflated_wl.nc4c',
+            'test_echam_spectral-deflated_ws.nc4c',
+            'test_echam_spectral-deflated_wind10_wl_ws.nc4c'
+        ]
```

Comment:
The old formatting was also correct. I suggest leaving it as it was.

Reply:
The last list item exceeded 79 characters that's why I did it.